shared/data/constants.js

```diff
@@ -97,6 +97,20 @@ module.exports = {
             "source": "external"
         }
     ],
+    "UserAgentLists": [
+        {
+            "name": "agents",
+            "url": "https://staticcdn.duckduckgo.com/useragents/random_useragent.json",
+            "format": "json",
+            "source": "external"
+        },
+        {
+            "name": "excludeList",
+            "url": "https://staticcdn.duckduckgo.com/useragents/useragent_excludes.json",
+            "format": "text",
```

Comment:
why it's text and not json?

Reply:
great catch, forgot to update that when I changed the list format to json :)